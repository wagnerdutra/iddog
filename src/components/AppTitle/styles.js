import styled from 'styled-components';

const Title = styled.h1`
  letter-spacing: 10px;
  font-weight: normal;
  white-space: nowrap;

  span {
    font-weight: bold;
  }
`;

export { Title };
