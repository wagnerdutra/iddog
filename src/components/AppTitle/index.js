import React from 'react';

import { Title } from './styles';

const AppTitle = () => (
  <Title>
    THE
    <span> IDDOG</span>
  </Title>
);

export default AppTitle;
