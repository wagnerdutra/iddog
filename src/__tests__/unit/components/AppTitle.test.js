import React from 'react';

import { render } from 'react-testing-library';

import AppTitle from 'Components/AppTitle';

describe('<AppTitle />', () => {
  it('should render correctly', () => {
    const { container } = render(<AppTitle />);
    expect(container).toMatchSnapshot();
  });
});
