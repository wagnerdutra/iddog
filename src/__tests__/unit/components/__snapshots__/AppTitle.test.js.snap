// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppTitle /> should render correctly 1`] = `
.c0 {
  -webkit-letter-spacing: 10px;
  -moz-letter-spacing: 10px;
  -ms-letter-spacing: 10px;
  letter-spacing: 10px;
  font-weight: normal;
  white-space: nowrap;
}

.c0 span {
  font-weight: bold;
}

<div>
  <h1
    class="c0"
  >
    THE
    <span>
       IDDOG
    </span>
  </h1>
</div>
`;
